src/components/backend-ai-webui.ts

```diff
@@ -1237,6 +1244,27 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
     return name;
   }
 
+  _getAccessSecretKeys(accessKey) {
+    const fields = ['access_key', 'secret_key'];
+    return globalThis.backendaiclient.keypair.info(accessKey, fields);
+  }
+
+  async _showAccessSecretKeys() {
```

Comment:
As I mentioned at the description of this issue, `user id`(email address) can have more than one keypair(credential). That is, the relation between `user id` and keypair(access key / secret key) is 1 : N. 
This is a screenshot for a better understanding.
<img width="1159" alt="multiple-keypair-in-a-single-user-id" src="https://user-images.githubusercontent.com/46954439/125656173-6fdf4c85-8dcd-4441-94e7-72477c810198.png">




Although default settings and mostly we provides `user id` and keypairs as 1:1, but we'd better show all of access key and secret key in the information panel.
> Hint: To see how user id has an relationship between multiple keypairs, please follow the guide below.
> 1. Check out the Users menu in the Administration section in the sidebar. 
> 2. Click `CREDENTIALS` tab in the page.
> 3. Click `ADD CREDENTIAL` button in the right side of the table.
> 4. Input the account of superadmin in the  User ID field and Click ADD button at the bottom of the dialog.
> 5. Check the node SDK function that requests and receives keypair information. (👉 [LINK](https://github.com/lablup/backend.ai-webui/blob/9df0af1cd719aba3275d61f5f493f8911ce8ac17/src/lib/backend.ai-client-node.ts#L2000-L2110))

Reply:
As suggested, access keys and secret keys are listed in a grid table considering 1:N relationship of user id and keys.

![image](https://user-images.githubusercontent.com/49114533/125912388-7194efb2-48fc-449e-8f12-98c17d6d9af5.png)